yaml/landuse.yaml

```diff
@@ -339,12 +339,14 @@ filters:
     output:
       <<: *output_properties
       kind: forest
+      kind_detail: &woods_detail { col: leaf_type }
       tier: 2
   - filter: { landuse: forest }
     min_zoom: { max: [ 10, *tier2_min_zoom ] }
     output:
       <<: *output_properties
       kind: forest
+      kind_detail: *woods_detail
```

Comment:
While this may be technically correct YAML, it's doesn't follow our convention.

Instead of declaring the YAML global on line 342 with `kind_detail: &woods_detail { col: leaf_type }` instead add to the globals section around line 134 like:

```
- &leaftype_kind_detail
    kind_detail: { col: leaf_type }
```

and then the usage down below (like 342, 349, etc) would be:

```
      <<: * leaftype_kind_detail
```